src/grid/Col.js

```diff
@@ -1,30 +1,41 @@
 import React, { PropTypes } from 'react';
-import { View, TouchableOpacity } from 'react-native';
+import { View, StyleSheet, TouchableOpacity } from 'react-native';
 
 const Col = (props) => {
-  const {style, size, onPress, activeOpacity} = props;
+  const {containerStyle, size, onPress, activeOpacity} = props;
 
-  const styles = {
-    flex: (size) ? size : (style && style.width) ? 0 : 1,
-    flexDirection: 'column',
-    ...style,
-  };
+  const styles = StyleSheet.create({
```

Comment:
@Monte9 The reason to have styles inside the `Col` object is because we need access to the props `containerStyle` and `containerStyle.width`